client-go/controller/models/apps/apps_test.go

```diff
@@ -291,7 +291,7 @@ func TestAppsList(t *testing.T) {
 
 	client := client.Client{HTTPClient: httpClient, ControllerURL: *u, Token: "abc"}
 
-	actual, err := List(&client)
+	actual, err := List(&client, 100)
```

Comment:
Good Idea! I implemented it. In this particular location these are just the tests and I just gave a random value as it doesn't affect the response.